lib/OpenQA/WebAPI/Controller/Step.pm

```diff
@@ -262,7 +262,7 @@ sub edit ($self) {
             tags => $tags,
             default_needle => $default_needle,
             error_messages => \@error_messages,
-            git_enabled => ($app->config->{global}->{scm} // '') eq 'git',
+            git_enabled => config_autocommit_enabled($app->config),
```

Comment:
I believe it 'd better to rename also the variable here.

Reply:
You mean `git_enabled`? mm, I guess we could.

Reply:
done.